docs/topics/mpp/multiplatform.md

```diff
@@ -7,68 +7,70 @@
+* To interop with platforms, use platform-specific versions of Kotlin. **Platform-specific versions of Kotlin** 
 (Kotlin/JVM, Kotlin/JS, Kotlin/Native) include extensions to the Kotlin language, and platform-specific libraries and tools. 
-*   Through these platforms you can access the **platform native code** (JVM, JS, and Native) and leverage all native
+* Through these platforms you can access the **platform native code** (JVM, JS, and Native) and leverage all native
  capabilities.
 
+## Code sharing between platforms
+
 With Kotlin Multiplatform, spend less time on writing and maintaining the same code for [different platforms](mpp-supported-platforms.md)
  – just share it using the mechanisms Kotlin provides:
 
 * [Share code among all platforms used in your project](mpp-share-on-platforms.md#share-code-on-all-platforms). Use it for sharing the common 
-business logic that applies to all platforms. 
+business logic that applies to all platforms:
      
     ![Code shared for all platforms](flat-structure.png)
     
 * [Share code among some platforms](mpp-share-on-platforms.md#share-code-on-similar-platforms) included in your project but not all. Do this 
-when you can reuse much of the code in similar platforms.  
+when you can reuse much of the code in similar platforms:
     
     ![Hierarchical structure](hierarchical-structure.png)
 
     ![Code shared for iOS targets](iosmain-hierarchy.png){width=400}
 
-If you need to access platform-specific APIs from the shared code, use the Kotlin mechanism of [expected and actual 
+* If you need to access platform-specific APIs from the shared code, use the Kotlin mechanism of [expected and actual 
 declarations](mpp-connect-to-apis.md).
 
-With this mechanism, a common source set defines an _expected declaration_, and platform source sets must provide the 
+  With this mechanism, a common source set defines an _expected declaration_, and platform source sets must provide the 
 _actual declaration_ that corresponds to the expected declaration. This works for most Kotlin declarations, such as 
 functions, classes, interfaces, enumerations, properties, and annotations.
 
-![Expect and actual declarations](expect-actual.png){width=700}
+    ![Expect and actual declarations](expect-actual.png){width=700}
 
-```kotlin
-//Common
-expect fun randomUUID(): String
-```
+    ```kotlin
+    // Common
+    expect fun randomUUID(): String
+    ```
 
-```kotlin
-//Android
-import java.util.*
-actual fun randomUUID() = UUID.randomUUID().toString()
-```
+    ```kotlin
+    // Android
+    import java.util.*
+    actual fun randomUUID() = UUID.randomUUID().toString()
+    ```
 
-```kotlin
-//iOS
-import platform.Foundation.NSUUID
-actual fun randomUUID(): String = NSUUID().UUIDString()
-```
+    ```kotlin
+    // iOS
+    import platform.Foundation.NSUUID
+    actual fun randomUUID(): String = NSUUID().UUIDString()
+    ```
 
-## Use cases
+## Kotlin Multiplatform use cases
```

Comment:
Seems unnecessary under the general 'Kotlin Multiplatform' tiltle. Or it's vital for SEO?

Reply:
My thoughts were about SEO